src/library-authoring/add-content/messages.ts

```diff
@@ -51,6 +66,11 @@ const messages = defineMessages({
     defaultMessage: 'There was an error creating the content.',
     description: 'Message when creation of content in library is on error',
   },
+  successAssociateComponentMessage: {
+    id: 'course-authoring.library-authoring.associate-collection-content.success.text',
+    defaultMessage: 'Content linked successfully.',
```

Comment:
Not sure how I feel about using the term "linking" here, since it's very different than linking content to a course. No action needed now, just a potential place for refinement post-Sumac.

Reply:
+1